src/single_cell/ugbio_single_cell/collect_statistics.py

```diff
@@ -208,11 +208,15 @@ def extract_statistics_table(h5_file: Path):
 
     with pd.HDFStore(h5_file, "r") as store:
         # number of Input Reads
-        num_input_reads = store[H5Keys.TRIMMER_STATS.value]["num input reads"].to_numpy()[0]
+        num_input_reads_list = store[H5Keys.TRIMMER_STATS.value]["num input reads"].to_numpy()
+        num_input_reads = next((x for x in num_input_reads_list if x != 0), None)
+        if num_input_reads is None:
+            raise ValueError("Number of input reads in trimmer statistics is not available.")
```

Comment:
do we want the task to fail in this case?
if we do, can you please add a test for this use case?